temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -109,6 +114,10 @@ message NamespaceSpec {
     // Optional, default is unset.
     CodecServerSpec codec_server = 6;
 
+    // The disaster recovery configuration for the namespace.
+    // temporal:versioning:min_version=2025-01-01-00
```

Comment:
@anekkanti do you know if I need to change this version to v0.4.0?

Reply:
Yes you do need to change the version to `v0.4.0`
Thanks!